suite-native/module-accounts-import/src/utils.ts

```diff
@@ -0,0 +1,14 @@
+import { AccountType, NetworkSymbol } from '@suite-common/wallet-config';
+import { getXpubOrDescriptorInfo } from '@trezor/utxo-lib';
+
+import { paymentTypeToAccountType } from './constants';
+
+export const getAccountTypeFromDescriptor = (
```

Comment:
no need to have utils file, please place this to `accountsImportThunk`. Always keep this utils functions in close to place where they are used unless they are needed in multiple places.

Reply:
Moved in 0370f3e50.